src/overlays/fbdemos/ovl_fbdemo_wipe4/z_fbdemo_wipe4.c

```diff
@@ -1,11 +1,21 @@
-/*
- * File: z_fbdemo_wipe4.c
+/**
+ * @file z_fbdemo_wipe4.c
  * Overlay: ovl_fbdemo_wipe4
  * Description:
+ *
+ * @note The bug in TransitionWipe4_Draw() makes this transition do nothing apart from increase its timer until it is
+ * finished.
  */
 
 #include "z_fbdemo_wipe4.h"
 
+#define THIS ((TransitionWipe4*)thisx)
+
+// TODO: determine how generic these macros are. Requires rest of the fbdemos decompiled.
+#define TW4_SET_PARAMS (1 << 7)
```

Comment:
Based on what I've learned in play, this is general (though only used for Wipe3 and Wipe4)

Reply:
I think I'd be inclined to leave this for now and have a general cleanup PR when everything's in, since we don't know when that's going to be.

Reply:
That is fine. If this gets in before play I'll update it.